src/NuGet.CommandLine/Commands/RestoreCommand.cs

```diff
@@ -324,7 +323,7 @@ private void ReadSettings(PackageRestoreInputs packageRestoreInputs)
                 {
                     var message = string.Format(
                         CultureInfo.CurrentCulture,
-                        "RestoreCommandFileNotFound",
+                        LocalizedResourceManager.GetString("RestoreCommandFileNotFound"),
```

Comment:
Please use nameof(NuGetResources.RestoreCommandFileNotFound).

I don't know why InvalidOperationException is thrown here. Probably we need to retain it. But, CommandLineException takes in a format and applies CurrentCulture and so on.
Instead of 2 lines, it could have been 
'throw new CommandLineException(LocalizedResourceManager.GetString(....), packageReferenceFile);'